src/components/Notification/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {useEffect, useState} from 'react';
+import {useAppDispatch, useAppSelector} from '@hooks/hooks';
+import {Text, TouchableOpacity} from 'react-native';
+import AntDesign from 'react-native-vector-icons/AntDesign';
+import {
+  Container,
+  TextBlock,
+  Wrapper,
+} from '@src/components/Notification/styles';
+import {Color} from '@theme/colors';
+import {hideAppMessage, setAppLastMessage} from '@src/bll/appReducer';
+
+export const Notification = () => {
+  const dispatch = useAppDispatch();
+  const messages = useAppSelector(state => state.app.messages);
+
+  const [timerId, setTimerId] = useState<ReturnType<typeof setInterval>>();
+
+  const setIsOpen = (id: number) => {
+    dispatch(hideAppMessage(id));
+  };
+
+  const stopTimer = () => {
+    clearInterval(timerId);
+  };
+
+  const startTimer = () => {
+    stopTimer();
+    const id: ReturnType<typeof setInterval> = setInterval(() => {
+      dispatch(setAppLastMessage());
+    }, 2900);
+    setTimerId(id);
+  };
+
+  useEffect(() => {
+    messages.length ? startTimer() : stopTimer();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why does it need here?

Reply:
ESlint wants me to add 'startTimer' and 'stopTimer' as dependencies. These dependencies are not needed there. When I add them, the code stops working.